src/zkevm_specs/evm/table.py

```diff
@@ -408,10 +418,18 @@ class CopyTableRow(TableRow):
     src_addr_end: FQ
     dst_addr: FQ
     length: FQ
+    value_rlc: FQ
     rw_counter: FQ
     rwc_inc: FQ
 
 
+@dataclass(frozen=True)
+class KeccakTableRow(TableRow):
+    idx: FQ
+    hash_rlc: FQ
+    value_rlc: FQ
```

Comment:
From my understanding, the keccak lookup has the following columns:
- is_enabled (should be 1 for a lookup)
- input_rlc
- length
- hash_rlc

Instead of `length` you defined `idx`.  Why is that?  I think this type should only use the column meanings used for lookup (without paying attention to the column meaning in the keccak table intermediate rows not used for lookup)

Reply:
I hadn't seen these docs before. But yes, that helps. And I have fixed at least the Keccak lookup in the PR.